### ISSUE TITLE
UART bootloader

### ISSUE THREAD

**Issue by Xplorer001** [Ameba8195/Arduino#1]
I see that currently Arduino board configuration uses SWD interface for programming. Is there a working UART boot loader available for this module?

**Comment by Ameba8195** [comment]
No, the log uart is too slow compare to SWD. It would takes a long time to flash image. We have noticed this feature and will make other improvement.
